orchestration/dagster_orchestration/hca_manage/soft_delete.py

```diff
@@ -24,6 +34,83 @@ def run(arguments: Optional[list[str]] = None) -> None:
 
 
 def soft_delete(args: argparse.Namespace, host: str) -> None:
-    hca = HcaManage(environment=args.env, data_repo_client=get_api_client(host=host),
-                    project=args.project, dataset=args.dataset)
+    hca = SoftDeleteManager(environment=args.env, data_repo_client=get_api_client(host=host),
+                            project=args.project, dataset=args.dataset)
```

Comment:
nit: arguments should either all be on one line or each have their own line
```suggestion
    hca = SoftDeleteManager(
        environment=args.env,
        data_repo_client=get_api_client(host=host),
        project=args.project,
        dataset=args.dataset
   )
```